refinery/ui/source/js/user-file-browser/services/user-file.spec.js

```diff
@@ -160,7 +160,7 @@
             .expectGET(
                 settings.appRoot +
                 settings.refineryApiV2 +
-                '/user/files/?fq=&limit=100'
+                '/user/files/?fq=&limit=100&sort='
```

Comment:
actually, now that I look at it, that is already being done with `fakeResponse`.